### PR TITLE
add doc string for functions

### PR DESCRIPTION
**Pull request by Liyuan-Chen-1024**
add doc string for functions

### CODE REVIEW

**Comment by ericspod** [comment]
Todo:
* fill in more param tags for places where they're missing
* fix parram typo
* describe a little more what the fast csv reader is doing, ie. why is it fast

File: exetera/core/operations.py [hunk 1]
```diff
@@ -2793,7 +2793,9 @@ def raiseNumericException(exception_message, exception_args):
 
 def transform_int(column_inds, column_vals, column_offsets, col_idx,
                     written_row_count, invalid_value, validation_mode, data_type, field_name):
-
+    """
```
Comment (atbenmurray) [inline exetera/core/operations.py:2796]: Tranform -> Transform

File: exetera/core/operations.py [hunk 2]
```diff
@@ -2793,7 +2793,9 @@ def raiseNumericException(exception_message, exception_args):
...
+    Tranform int method for numeric importer in field_importer.py
```
Comment (atbenmurray) [inline exetera/core/operations.py:2797]: Tranform -> Transform

File: exetera/core/operations.py [hunk 3]
```diff
@@ -2836,7 +2838,9 @@ def transform_int(column_inds, column_vals, column_offsets, col_idx,
 
 def transform_float(column_inds, column_vals, column_offsets, col_idx,
                       written_row_count, invalid_value, validation_mode, data_type, field_name):
-
+    """
```
Comment (atbenmurray) [inline exetera/core/operations.py:2841]: Tranform -> Transform

File: exetera/core/operations.py [hunk 4]
```diff
@@ -2894,6 +2898,9 @@ def transform_to_values(column_inds, column_vals, column_offsets, col_idx, writt
 @njit
 def fixed_string_transform(column_inds, column_vals, column_offsets, col_idx, written_row_count,
                            strlen, memory):
+    """
```
Comment (atbenmurray) [inline exetera/core/operations.py:2901]: Tranform -> Transform

File: exetera/core/field_importers.py [hunk 5]
```diff
@@ -114,6 +114,16 @@ def __init__(self, create_day_field=False, create_flag_field=False):
 #============= Field Importers ============
 
 class CategoricalImporter:
+    """
+    Importer for Categorical field.
```
Comment (atbenmurray) [inline exetera/core/field_importers.py:118]: Please say something about how the importers are used and whether end users are expected to use them directly for each importer

**Review by ericspod — APPROVED** [review]

**Pull request merged**